neqo-transport/src/frame.rs

```diff
@@ -594,9 +594,14 @@ impl Frame {
             }),
             FRAME_TYPE_CRYPTO => {
                 let o = dv!(dec);
+                let l = dv!(dec);
```

Comment:
You can still use decode_vvec here.  Just check that `o + vec.len() < ((1<<62)-1)`.  You can do that before you call to_vec() and do the copy.

Reply:
Sure, I see. Thanks for letting me know.

Reply:
Hi @martinthomson ,

With the method you mentioned, I need to convert from usize to u64 `o + vec.len() as u64 < ((1<<62)-1)`. Though most likely this should be safe for quite a long while, but it is not really future proofing.

Is there any better way to handle this? Please help.

Thanks and Best Regards

Reply:
The conversion should be ok. Packets don't get that big. As long as we check the conversion, we are ok. So far, when it comes to lengths, our policy is that we will accept up to whatever usize AND u64 can hold. Mostly that is the same thing, but even if that is only 2^32, that's a lot.

Use u64::try_from rather than `as`.

Reply:
Got it, thanks a lot.